utils/kubernetes/apply-helm-chart.go

```diff
@@ -74,10 +74,12 @@ type HelmEntries map[string][]HelmEntryMetadata
 // HelmEntryMetadata is the struct for holding the metadata
 // associated with a helm repositories' entry
 type HelmEntryMetadata struct {
-	APIVersion string `yaml:"apiVersion"`
-	AppVersion string `yaml:"appVersion"`
-	Name       string `yaml:"name"`
-	Version    string `yaml:"version"`
+	APIVersion  string `yaml:"apiVersion"`
```

Comment:
Is there anyway for MeshKit to reference the Helm Controller model, using values ComponentDefinitions (granted that we wouldn't support all types of Helm connections immediately)?

Reply:
This struct represents the index.yml and is used when deploying/fetching the helm charts. Referencing the Helm Controller model wouldn't serve the purpose here.

Reply:
It would serve that purpose, but also many other purposes. I'm asking if we can reference that component, and only expose a single field.